### PR TITLE
99 rename daemon relocate crate

### PR DESCRIPTION
**Pull request by BillyWooo**
resovles #99

### CODE REVIEW

File: litentry/core/stf-task-handler/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [package]
-name = "itc-extrinsic-request-daemon"
+name = "itc-stf-task-handler"
```
Comment (Kailai-Wang) [inline litentry/core/stf-task-handler/Cargo.toml]: We have a mixture of `itc-` and `lc-`

File: enclave-runtime/src/lib.rs [hunk 2]
```diff
@@ -69,11 +69,11 @@ use substrate_api_client::{compose_extrinsic_offline, ExtrinsicParams};
 
 mod attestation;
 mod empty_impls;
-mod extrinsic_daemon;
 mod global_components;
 mod initialization;
 mod ipfs;
 mod ocall;
+mod stf_task_thread;
```
Comment (Kailai-Wang) [inline enclave-runtime/src/lib.rs:76]: can we move `stf_task_thread` to `stf-task-handler` crate?

File: enclave-runtime/src/stf_task_thread.rs [hunk 3]
```diff
@@ -117,7 +117,7 @@ fn run_extrinsic_request_daemon_internal() -> Result<()> {
 	let stf_enclave_signer =
 		Arc::new(EnclaveStfEnclaveSigner::new(state_observer, ocall_api, shielding_key_repository));
 
-	let request_context = RequestContext::new(
+	let request_context = VerifyContext::new(
```
Comment (Kailai-Wang) [inline enclave-runtime/src/stf_task_thread.rs:120]: Can we have a `stf_task_receiver.rs` to handle the stf-tasks?
Then inside the thread it should as simple as launching the receiver

File: enclave-runtime/src/lib.rs (hunk 2 above)
Comment (BillyWooo) [reply enclave-runtime/src/lib.rs:76]: This is the problem I try to recall in our discussion. There is recursive include from enclave-runtime to litentry/* , then back to enclave-runtime.

File: litentry/core/stf-task-handler/Cargo.toml (hunk 1 above)
Comment (BillyWooo) [reply litentry/core/stf-task-handler/Cargo.toml]: fixed.

File: enclave-runtime/src/lib.rs (hunk 2 above)
Comment (Kailai-Wang) [reply enclave-runtime/src/lib.rs:76]: Which one exactly?

File: enclave-runtime/src/stf_task_thread.rs (hunk 3 above)
Comment (BillyWooo) [reply enclave-runtime/src/stf_task_thread.rs:120]: I have thought about creating a `stf_task_receiver.rs` and move it to `litentry/core/stf_task_handler/`. But it came to the same problem as above of recursive include. 
Or do you mean simply split into two files, but still sit inside `enclave-runtime` ?

File: enclave-runtime/src/lib.rs (hunk 2 above)
Comment (BillyWooo) [reply enclave-runtime/src/lib.rs:76]: Currently, inside `enclave-runtime/Cargo.toml` it depends on `lc-stf-task-handler` and `lc-identity-verify-handler`. `stf_task_thread` also depends on the `error` inside `enclave-runtime`. If I move `stf_task_thread` into `litentry/core` folder, recursive include happened.

File: litentry/core/stf-task-handler/src/stf_task_sender.rs [hunk 4]
```diff
@@ -59,18 +59,18 @@ impl SendXTRequest for XTRequestSender {
 		// Acquire lock on extrinsic sender
 		let mutex_guard = GLOBAL_XT_DAEMON.lock().map_err(|_| Error::MutexAccess)?;
```
Comment (Kailai-Wang) [inline litentry/core/stf-task-handler/src/stf_task_sender.rs:60]: We still have some `XTDaemon` stuff 😂  (and below)

File: enclave-runtime/src/lib.rs (hunk 2 above)
Comment (Kailai-Wang) [reply enclave-runtime/src/lib.rs:76]: I see, thanks. Let's then keep `stf_task_thread` in `enclave-runtime` but have a separate `stf_task_receiver.rs` under `litentry/core/stf-task-handler`. It should only contain the loop and doesn't need to refer to the `Error` type

**Pull request merged**